config/pipeline.yaml

```diff
@@ -910,6 +910,28 @@ jobs:
       job_timeout: 10
     kcidb_test_suite: kselftest.cpufreq
 
+  kselftest-cpufreq-hibernate:
+    <<: *kselftest-job
+    params:
+      <<: *kselftest-params
+      collections: cpufreq
+      env: 'KSELFTEST_MAIN_SH_ARGS="-t hibernate_rtc"'
+    rules:
```

Comment:
Maybe we want to run this test on all the trees and not just kernelci. I may not have enough context for this. So what do you think?